src/pages/Lend/Filter/FilterRangeSlider.vue

```diff
@@ -100,16 +98,65 @@ export default {
 			default: null,
 			required: true
 		},
+		range: {
+			type: Object,
+			default: null,
+			required: true
+		},
+		refine: {
+			type: Function,
+			default: null,
+			required: true
+		},
+	},
+	computed: {
+		// set label for slider
+		rangeLabel() {
+			let value = this.maximumLabel;
+
+			if (this.selectedRangeValue === this.maximum) {
+				value = this.maximumLabel;
+
+			// minimum value
+			} else if (this.selectedRangeValue === this.minimum) {
+				if (this.minimumValueLabel === false) {
+					value = this.minimumLabel;
+				} else {
+					value = `${this.selectedRangeValue}${this.minimumLabel}`;
+				}
+
+			// values in between min/max value
+			} else {
+				value = `${this.selectedRangeValue}${this.inbetweenLabel}`;
+			}
+
+			return value;
+		},
+	},
+	watch: {
+		// upon deletion of refinement (filter menu closed) update range: label, value and style (liner gradient)
+		currentRefinement(data) {
+			if (this.filterMenuOpen === false) {
```

Comment:
This is confusing me...Not sure why we would want to do anything b/c you closed the Filter drop down.

Reply:
@mcstover It's when a user removes a facet when the filter menu is closed; the slider, label and gradient (gray / green) have to be updated.

Reply:
oh ok, thanks, that makes sense.

Reply:
Let's discuss this `watch` over our chat tomorrow. This runs 8 times every time we set or unset a filter. Is there a way to replace this functionality by hooking into the `transformItems` method?

Reply:
@ksekhri Unfortunately, `ais-range-input` does not have `transformItems`

Reply:
@ksekhri I've added this computed method. It's somewhat a hack, but it does work and is not as called as often as the watch method:

```
		// set slider position
		sliderPosition() {
			//	eslint-disable-next-line max-len
			if (this.filterMenuOpen === false && this.currentRefinement.min === null && this.currentRefinement.max === null) {
				this.setSliderValue();
			}

			return '';
		},
```